.github/workflows/run-test-and-format.yml

```diff
@@ -0,0 +1,53 @@
+name: Run tests & format
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
      - '**'
```
Waarom alleen main?

Reply:
`workflow_dispatch:` Hiermee kun je de actie handmatig uitvoeren via het `Actions`, mocht dat nodig zijn om wat voor reden

`pull_request:` Triggerd bij elke pull request of push in de PR, en alle code wordt altijd naar de main branch gepusht. Wat ik vaak zie, is dat de code naar main gaat, wat eigenlijk een soort van productie- of releasecode is. En vaak word dat nog 1x door een GH Action heen gehaald.

Zie screenshot voor `pull_request` demo

![CleanShot 2024-08-12 at 17 06 34](https://github.com/user-attachments/assets/093623f1-b722-4f0e-83be-32d01caa7269)